src/pages/Facility/settings/organizations/FacilityOrganizationUsers.tsx

```diff
@@ -159,17 +160,29 @@ export default function FacilityOrganizationUsers({ id, facilityId }: Props) {
                         </div>
                       </div>
 
-                      <div className="mt-2 -mx-2 -mb-2 sm:-mx-4 sm:-mb-4 rounded-md py-4 px-4 bg-gray-50 flex justify-end">
+                      <div className="mt-2 -mx-2 -mb-2 sm:-mx-4 sm:-mb-4 rounded-md py-4 px-4 bg-gray-50 flex justify-end gap-2">
                         <EditFacilityUserRoleSheet
                           facilityId={facilityId}
                           organizationId={id}
                           userRole={userRole}
                           trigger={
                             <Button variant="outline" size="sm">
-                              <span>{t("see_details")}</span>
+                              <span>{t("edit_role")}</span>
                             </Button>
                           }
                         />
+                        <Button variant="outline" size="sm">
```

Comment:
asChild prop.

Also move this button out of the trigger. Only Edit role button should be in trigger.